app/views/downloads/download_linux.html.erb

```diff
@@ -56,5 +56,63 @@
 
   <h3>Slitaz</h3>
   <code>$ tazpkg get-install git</code>
+  
```

Comment:
we should probably add some space between both 'sections': https://git-scm-pr-1495.herokuapp.com/download/linux

Reply:
Sure, that will give more space between the two.